grails-app/i18n/messages.properties

```diff
@@ -2,3 +2,776 @@
 map.controls.ta_grid_color_mode.linear = Linear
 map.controls.ta_grid_color_mode.logscale = Log-scale
 map.controls.ta_grid_color_mode.quantile = Quantile
+
+generic.button.cancel = Cancel
+generic.button.close = Close
+generic.button.yes = Yes
+generic.button.no = No
+false = No
```

Comment:
use generic.false or general.false or smth. It easier to search over the project for occurrences